superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -1003,6 +1010,46 @@ export const controls = {
     'relative to the time granularity selected'),
   },
 
+  cell_size: {
+    type: 'TextControl',
+    isInt: true,
+    default: 10,
+    validators: [v.integer],
+    renderTrigger: true,
+    label: t('Cell Size'),
+    description: t('The size of the square cell, in pixels'),
+  },
+
+  cell_padding: {
+    type: 'TextControl',
+    isInt: true,
+    validators: [v.integer],
+    renderTrigger: true,
+    default: 2,
+    label: t('Cell Padding'),
+    description: t('The distance between cells, in pixels'),
+  },
+
+  cell_radius: {
+    type: 'TextControl',
+    isInt: true,
+    validators: [v.integer],
+    renderTrigger: true,
+    default: 2,
```

Comment:
Personally, I'd leave this at zero — the round borders look really ugly to me, unless they're full circles. Maybe someone with a better design eye than me can comment on this.

Reply:
Agreed, I set `cell_radius` to `0` just now. This is `cell_padding` though, think I should set this to 0 as well? There's this annoying ghost black spots that happens in the corners 👻 
<img width="671" alt="screen shot 2018-04-11 at 3 11 28 pm" src="https://user-images.githubusercontent.com/487433/38650726-4d9fd4c8-3db2-11e8-997e-7a24a81740ba.png">

Reply:
Argh, the black spots keep moving! :-P

I think if we reduce to a small value they should go away, no? If not, I'm fine with 0.

Reply:
0 will remove them, changing now